apps/storybook/src/Table.stories.tsx

```diff
@@ -52,55 +52,9 @@ import {
 export default {
   title: 'Core/Table',
   component: Table,
-  args: {
-    data: [
-      { name: 'Name1', description: 'Description1' },
-      { name: 'Name2', description: 'Description2' },
-      { name: 'Name3', description: 'Description3' },
-    ],
-    emptyTableContent: 'No data.',
-    density: 'default',
-    emptyFilteredTableContent: 'No results found. Clear or try another filter.',
-  },
-  argTypes: {
-    columns: { control: { disable: true } },
-    isSelectable: { control: { disable: true } },
-    selectionMode: { control: { disable: true } },
-    style: { control: { disable: true } },
-    className: { control: { disable: true } },
-    id: { control: { disable: true } },
-    initialState: { table: { disable: true } },
-    stateReducer: { table: { disable: true } },
-    useControlledState: { table: { disable: true } },
-    defaultColumn: { table: { disable: true } },
-    getSubRows: { table: { disable: true } },
-    getRowId: { table: { disable: true } },
-    manualRowSelectedKey: { table: { disable: true } },
-    autoResetSelectedRows: { table: { disable: true } },
-    selectSubRows: { table: { disable: true } },
-    manualSortBy: { table: { disable: true } },
-    defaultCanSort: { table: { disable: true } },
-    disableMultiSort: { table: { disable: true } },
-    isMultiSortEvent: { table: { disable: true } },
-    maxMultiSortColCount: { table: { disable: true } },
-    disableSortRemove: { table: { disable: true } },
-    disabledMultiRemove: { table: { disable: true } },
-    orderByFn: { table: { disable: true } },
-    sortTypes: { table: { disable: true } },
-    autoResetSortBy: { table: { disable: true } },
-    autoResetHiddenColumns: { table: { disable: true } },
-    autoResetFilters: { table: { disable: true } },
-    filterTypes: { table: { disable: true } },
-    defaultCanFilter: { table: { disable: true } },
-    manualFilters: { table: { disable: true } },
-    paginateExpandedRows: { table: { disable: true } },
-    expandSubRows: { table: { disable: true } },
-    autoResetExpanded: { table: { disable: true } },
-    manualExpandedKey: { table: { disable: true } },
-  },
 } as Meta<TableProps>;
 
-export const Basic: Story<Partial<TableProps>> = (args) => {
+export const Basic: Story<Partial<TableProps>> = () => {
```

Comment:
The type `Story<Partial<TableProps>>` can also be removed

Reply:
https://github.com/iTwin/iTwinUI/pull/1543/commits/45ca69b7b9a46a7970122c711495b0f39bedd826: Removed `Story<Partial<TableProps>>`

https://github.com/iTwin/iTwinUI/pull/1543/commits/a5bf4c7dad28171acc9c927d82508fe2f2cce684: I also removed `as Meta<TableProps>` since I believe that has something to do with assigning the correct type to `args`, and thus is no longer needed.